src/python/pants/backend/python/util_rules/pex_cli.py

```diff
@@ -202,6 +208,15 @@ async def setup_pex_cli_process(
     )
 
 
+def maybe_log_pex_stderr(stderr: bytes, pex_verbosity: int) -> None:
+    """Forward Pex's stderr to a Pants logger if conditions are met."""
+    log_output = stderr.decode()
+    if log_output and pex_verbosity > 0:
+        logger.info("%s", log_output)
+    elif log_output and "PEXWarning:" in log_output:
```

Comment:
What do you think about potentially being even more lax here: I would think showing more warnings is better than missing some, e.g. just check for `warning` (case insensitively)?

Reply:
I'm worried about making a wishy washy guess that someday might cause someone to have to think about unicode case folding in a whacky corner case.  I'd rather either have the `PEXWarning` check, or stderr unconditionally.